scripts/claimVestedTokens.js

```diff
@@ -0,0 +1,98 @@
+// ES5 style
+const config = require("./vesterConfig");
+const thorify = require("thorify").thorify;
+const Web3 = require("web3");
+const TreasuryVester = require(config.pathToTreasuryVesterJson);
+const Vex = require(config.pathToVEXJson);
+const assert = require('assert');
+
+let rpcUrl = null;
+let vexAddress = null;
+let vesterAddress = null;
+
+if (process.argv.length < 4) 
+{
+    console.error("Usage: node claimVestedTokens.js [mainnet|testnet] [address of Vester contract]");
+    process.exit(1);
+} 
+else
+{
+    if (process.argv[2] == "mainnet") 
+    {
+        rpcUrl = config.mainnetRpcUrl;
+        vexAddress = config.mainnetVexAddress;
+    }
+    else if (process.argv[2] == "testnet") 
+    {
+        rpcUrl = config.testnetRpcUrl;
+        vexAddress = config.testnetVexAddress;
+    }
+    else 
+    {
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
+
+    if(!Web3.utils.isAddress(process.argv[3]))
+    {
+        console.error("Invalid address for vester contract");
```

Comment:
Interesting, so each user has their own personal vester contract, got it

Reply:
Correct, each TreasuryVester is designed to vest for only one address. 

Everyone can claim for themselves with the script, although it'd be public service if you do it for everyone from 0x15f